ui/tests/helpers/components/mount-backend-form-helpers.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { fillIn, click } from '@ember/test-helpers';
+import { MOUNT_BACKEND_FORM } from 'vault/tests/helpers/components/mount-backend-form-selectors';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+
+export const mountBackend = async (type, path) => {
```

Comment:
this could also be used here https://github.com/hashicorp/vault/pull/28798/files#diff-16d5bba5607b47a048d3d2d21fc875237c3421d51f702956d7761240d43175d4R29-R31